BuildVuClient/BuildVu.py

```diff
@@ -144,6 +146,21 @@ def __upload(self, params):
 
         return response['uuid']
 
+    @staticmethod
+    def get_returned_error_message(error, response):
```

Comment:
Suggestion to make the method private, you would also have to update the usages (line 138,173, 187)

```suggestion
    def __get_returned_error_message(error, response):
```